### PR TITLE
Change agents summary no results

### PR DESCRIPTION
**Pull request by JuanGarriuz**
### Description

 Agent summary feature: when there are no agents, we want to hide the graph and show a message and a button indicating the user how to deploy an agent
 
### Issues Resolved

- https://github.com/wazuh/wazuh-dashboard-plugins/issues/7027

### Evidence

image

### Test

- Check overview with and without agents

### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: plugins/main/public/controllers/overview/components/stats.js [hunk 1]
```diff
@@ -103,11 +105,34 @@ export const Stats = withErrorBoundary(
...
+                          permissions={[
+                            { action: 'agent:create', resource: '*:*:*' },
+                          ]}
+                          iconType='plusInCircle'
+                          href={NavigationService.getInstance().getUrlForApp(
+                            endpointSummary.id,
+                            {
+                              path: `#${endpointSummary.redirectTo()}deploy`,
+                            },
+                          )}
```
Comment (guidomodarelli) [inline plugins/main/public/controllers/overview/components/stats.js:128]: Extract to a const

File: plugins/main/public/controllers/overview/components/__snapshots__/stats.test.tsx.snap
Comment (guidomodarelli) [inline plugins/main/public/controllers/overview/components/__snapshots__/stats.test.tsx.snap:1]: Is it necessary to remove the entire snapshot?

**Review by guidomodarelli — COMMENTED** [review]
TEST :green_circle: 
image

File: plugins/main/public/controllers/overview/components/stats.js [hunk 2]
```diff
@@ -103,11 +113,29 @@ export const Stats = withErrorBoundary(
...
-                  noDataTitle='No results'
-                  noDataMessage='No results were found.'
-                />
+                    )}
+                  />
+                ) : (
+                  !hasResults &&
+                  this.props !== undefined && (
+                    <EuiEmptyPrompt
+                      body={<p>Add agents to fleet to start monitoring</p>}
```
Comment (asteriscos) [inline plugins/main/public/controllers/overview/components/stats.js:122]: Please change the body description to:

```
body={
    <p>
      This instance has no agents registered.
      <br />
      Please deploy agents to begin monitoring your
      endpoints.
    </p>
  }
```

**Review by asteriscos — CHANGES_REQUESTED** [review]

**Review by asteriscos — APPROVED** [review]
CR: :green_circle: 
Test: :green_circle:

**Pull request merged**